esp/templates/program/newprogram_review.html

```diff
@@ -38,18 +38,21 @@ <h2>Program Creation Review</h2>
 <div id="catalog">
 <table align="center">
 <tr>
-<th>The following program modules will be added:</th>
+<th colspan="2">The following program modules will be added:</th>
 </tr>
-<tr><th class="materials">Module Name</th></tr>
+<tr><th class="materials" colspan="2">Module Name</th></tr>
 {% for item in modules %}
-    <tr><td>{{ item.0 }}</td></tr>
+    {% if forloop.counter0|divisibleby:"2" %}<tr><td style="text-align:center">
```

Comment:
What is this doing?  It seems super fragile.  Can you instead put a method on whatever object this is, and just call that, so it would be `{{ item.0.prettyName }}` or whatever?

Reply:
It puts the items in two columns, alternating so that it doesn't leave an unclosed tag if there's an odd number of items. Do you think it makes sense to make a separate method for modules to arrange themselves into two HTML columns?

Reply:
Oh, I see.  Maybe comment that, but inline is fine.  I meant what the body is doing -- the `word != "Program"` stuff.